pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/MetadataCacheConfig.java

```diff
@@ -46,5 +46,12 @@ public class MetadataCacheConfig {
      * A negative or zero value disables automatic expiration.
      */
     @Builder.Default
-    private final long expireAfterWriteMillis = 2 * DEFAULT_CACHE_REFRESH_TIME_MILLIS;
+    private final long expireAfterWriteMillis = 0L;
```

Comment:
Change the default value to `-1`. Use -1 to disable the expire maybe better to understand.